src/Chapter20.Tests/Listing20.11.ImplementingCustomAsynchronousMethod.Tests.cs

```diff
@@ -1,10 +1,8 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System;
 using System.Text.RegularExpressions;
-using System.Threading.Tasks;
-using AddisonWesley.Michaelis.EssentialCSharp.Chapter20.Listing20_09;
 
-namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter20.Listing20_11.Tests
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter20.Listing20_09.Tests
```

Comment:
Was this change intentional? Is there a reason this doesn't match the file name?

Reply:
The file name was going to get updated here: #246 but I can do it in this one instead just for continuity

Reply:
Ah, either is fine by me.